src/main/java/cs2103/v15_1j/jimjim/uifeedback/AliasListFeedback.java

```diff
@@ -0,0 +1,35 @@
+package cs2103.v15_1j.jimjim.uifeedback;
+
+import java.util.Map;
+import java.util.Map.Entry;
+
+import cs2103.v15_1j.jimjim.ui.MainViewController;
+
+public class AliasListFeedback implements UIFeedback {
+	Map<String, String> feedbackList;
+	
+	public AliasListFeedback(Map<String, String> feedbackList) {
+		this.feedbackList = feedbackList;
+	}
+	
+	public Map<String, String> getFeedbackList() {
+		return feedbackList;
+	}
+	
+	@Override
+	public void execute(MainViewController con) {
+		for (Entry<String, String> mapping : feedbackList.entrySet()) {
+			con.showNotification("\""+ mapping.getKey() + "\" has been added as an alias for "
```

Comment:
This message is misleading. Nothing is added as aliases by the command. Also by showing 1 notification for each alias, you are effectively only showing the last one because the previous ones will all disappear. Let's not implement this non-trivial feedback in this since it's for the UI to handle @SorataYuu